### ISSUE TITLE
JFS spools engine past 20% RPM

### ISSUE THREAD

**Issue by JMaverick16** [NikolaiVChr/f16#382]

**Comment by NikolaiVChr** [comment]
Confirmed.

However due to the way JSBsim jet engines works, its best to leave it at 20%
JSBsim engines spool up, and then stops, and thats when you can open cutoff.
As far as I know, there is no way to get it so there is a range where cutoff can be used.

![image](https://user-images.githubusercontent.com/3764194/143565429-00c9c684-069a-497f-b36a-68a5ebfb1aa3.png)

**Comment by Zaretto** [comment]
The only control is via the 

- <ignitionn1>
- <ignitionn2> 
- 

These default to 

- IgnitionN1 = 5.21; 
- IgnitionN2 = 25.18

As far as I can remember with the air start systems on the certified simulators that I've had access to generally there is an upper point to both n1% and n2% during an air start; and this typically varies with available air for the start (low pressure carts or having ECS / pax active will reduce this and may not be enough to get the rpm into the start window). However I don't think we can simulate this in JSBSim.

**Comment by Octal450** [comment]
@NikolaiVChr yes there is, I have spinning up to 29% on the MD-11, pilot can switch cutoff anywhere from 14% to 29% there.

Kind Regards,
Josh
